test/functionalTest/cases/2670_milliseconds_for_DateTime_attrs/2670_milliseconds_for_DateTime_attrs.test

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I would suggest to add a new .test to test miliseconds filtering capabilties. Something like this

```
01. Create E1 with attribute t=2020-05-10T01:02:03.123Z
02. Create E2 with attribute t=2020-05-10T01:02:03.456Z
03. Create E3 with attribute t=2020-05-10T01:02:03.789Z
04. Query for entities t>2020-05-10T01:02:03Z, get E1, E2, E3
05. Query for entities t<2020-05-10T01:02:03.500Z, get E1
06. Query for entities t=2020-05-10T01:02:03.200Z...2020-05-10T01:02:03.600Z, get E2
```

and a simila test based in sub filtering instead of GET filtering.

Reply:
Discussed with @kzangeli at skype, this will be done in a next PR. @fgalan will do it. NTC